Tests/SwiftDocCTests/DocumentationService/ConvertService/ConvertServiceTests.swift

```diff
@@ -1221,38 +1221,52 @@ class ConvertServiceTests: XCTestCase {
             assert: { renderNodes, referenceStore in
                 let referenceStore = try XCTUnwrap(referenceStore)
                 
-                XCTAssertEqual(
-                    Set(referenceStore.topics.keys.map(\.path)),
-                    [
-                        // Documentation extension files:
-                        "/documentation/MyKit",
-                        "/documentation/SideKit",
-                        "/documentation/MyKit/MyClass",
-                        "/documentation/MyKit/MyProtocol",
-                        "/documentation/SideKit/SideClass/init()",
-                        
-                        // Articles and tutorials:
-                        "/tutorials/TestOverview",
-                        "/tutorials/TestOverview/$volume",
-                        "/tutorials/TestOverview/Chapter-1",
-                        "/documentation/Test-Bundle/article",
-                        "/documentation/Test-Bundle/article2",
-                        "/documentation/Test-Bundle/article3",
-                        "/tutorials/Test-Bundle/TestTutorial",
-                        "/tutorials/Test-Bundle/TestTutorial2",
-                        "/tutorials/Test-Bundle/TestTutorialArticle",
-                        "/tutorials/Test-Bundle/TutorialMediaWithSpaces",
-                        "/documentation/Test-Bundle/Default-Code-Listing-Syntax",
-                    ]
-                )
-            
-                try self.assertReferenceStoreContains(
-                    referenceStore: referenceStore,
-                    topicPath: "/documentation/MyKit/MyClass",
-                    source: testBundleURL.appendingPathComponent("documentation/myclass.md"),
-                    title: "doc:MyKit/MyClass",
-                    isDocumentationExtensionContent: true
-                )
+                if LinkResolutionMigrationConfiguration.shouldUseHierarchyBasedLinkResolver {
+                    XCTAssertEqual(
+                        Set(referenceStore.topics.keys.map(\.path)),
+                        [
+                            // This bundle doesn't contain any symbol graphs so documentation extensions can't match with any symbols
```

Comment:
Should we have a separate test that verifies that documentation extensions that do have matching symbols are included in the reference store?

Reply:
@ethan-kusters, @franklinsch and I talked about this. The ConvertService relies on this old implementation detail but new code shouldn't rely on this.

I restored this behavior only for the ConvertService in https://github.com/apple/swift-docc/pull/565/commits/7033adf239fdebf13621b87e0814e61c85037c21 and filed https://github.com/apple/swift-docc/issues/567 to update the ConvertService to address this.